libs/pong/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@yari-internal/pong",
+  "version": "0.0.1",
+  "private": true,
+  "license": "MPL-2.0",
+  "type": "module",
+  "main": "index.js",
+  "dependencies": {
```

Comment:
Do we not have a `yarn.lock` file here intentionally?

Reply:
nope, just never had to run yarn here